src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -1996,6 +1996,12 @@ def test_aks_nodepool_snapshot(self, resource_group, resource_group_location):
         })
         print("The snapshot resource id %s " % snapshot_resource_id)
 
+        # update tags on nodepool snapshot
+        update_snapshot_cmd = 'aks nodepool snapshot update --resource-group {resource_group} --name {snapshot_name} --tags {tags} -o json'
```

Comment:
Where's `tags` defined? And please add some checks on the updated tags

Reply:
Fixed, thanks for catching I'm new to python and this repo 👍